src/alinea/adel/mtg_editions.py

```diff
@@ -377,17 +377,23 @@ def update_organ_elements(g, leaves=None, split=False):
                     [elt['area'] for elt in elts if
                      elt['label'].startswith('StemElement')])
 
-            # insert elts
-            if len(g.components(organ)) == 2:
+            # insert elts and updates at element scale
+            if len(g.components(organ)) == 2: # only top and base element
                 insert_elements(g, organ, elts)
             else:
                 for elt in elts:
                     label = elt.pop('label')
-                    vid_elt = find_label(label, g, organ)[0]
+                    vid_elt_value = find_label(label, g, organ) # Add HiddenElement XOR LeafElements and update the other one.
```

Comment:
Yes. However will not work correctly if several new component are added (will reverse the ordrer of insertion, as new component are all added before topElement)